# Identification of VUS Across the ATP7B Gene

## Authors <br>
Deniz Saltık 29381 <br>
Durmuş Erdem Kertmen 28071 <br>
Göktuğ Gençkaya 27888 <br>
Metin Alperen Balta 28231 <br>

## Introduction

P-type ATPases on the integral membrane are found to be important for maintaining homeostasis in eukaryotes and prokaryotes by regulating heavy-metals across the organism. P-type ATPases are involved in copper and iron metabolisms in eukaryotes like homo sapiens, and the functional changes in these ATPases cause Wilson’s disease and Melken disease (Forbes et al., 1998). This study is focused on specially the effects of mutations of the P-type ATPase 2, ATP7B that causes Wilson’s disease, which is caused by mutations on the ATP7B gene on the 13th chromosome. This autosomal recessive disease is seen on a person in every 35,000 people, and it affects the functioning of liver and neurological structures via accumulation of copper in several organs. <br> <br>
In the search of the genetic cause of the disease, more than 100 different mutations have been analyzed and identified according to being benign or pathogenic (Forbes et al., 1998). As some mutations of the gene ATP7B are classified according to their effects on the disease and the organism, some variants of the aforementioned gene are still unknown in terms of their significance of coding fully functional or defective ATPase 2 protein. The aim of this project is identifying the “Variance of Unknown Significance (VUS)” mutations via comparing the protein sequences of the related protein family and the unknown sequences and deciding the significance of these variants using conservation scores of the single amino acids. <br>


## Results
In this project, the gene ATP7B which causes the Wilson’s disease in homo sapiens were examined by analyzing it’s homologous sequences and comparing the entropy scores in order to identify the conserved amino acid positions among species carrying the homologous ATP7B gene. The conservation rate of these positions was calculated using the entropy calculation method, which allows for the quantification of amino acid changes of unknown significance. This method was adopted from a previously published source by Pei, J., & Grishin, N. V. (2020).<br>

As further described in the materials and methods section, the protein sequence 
of the gene ATP7B was used to retrieve the homologous sequences of the gene ATP7B by using BlastP. 1000 homologous sequences of the gene ATP7B were chosen and aligned by using the Muscle Alignment in the Mega11 software. A part of the alignment is displayed below, and the complete alignment can be viewed in Github. <br><br>

![Alignment](/alignment.jpeg) <font size="5">__Fig.1__ :The snippet of the alignment done using MEGA11. </font>

The file containing the alignment of 1000 homologous sequences of the gene ATP7B was used to create a phylogenetic tree in the Mega11 software after the homologous sequences were aligned. The phylogenetic tree that was created using the 1000 homologous sequences of the gene ATP7B can be seen below. It was created using the neighbor-joining method and displayed using FigTree. <br><br>

![Tree](/tree.png) <font size="5">__Fig.2__ :Phylogenetic tree of the 1000 sequences generated by Neighbor-joining method. </font>

The neighbor-joining method of building trees is based on distance, therefore the closer two species are to one another, the more closely related they are. It is a beneficial approach to comprehend the species' closest common ancestor and to categorize them.<br>

As it can observed in the Neighbor-joining tree above, the species closest to Homo sapiens are in the same clade and they are all coming from a common ancestor. The species closest to Homo sapiens have the same form of the ATP7B gene as homo sapiens, as expected before construing the phylogenetic tree. In the Neighbor-joining tree above, as the species are closer to each other in the clade, the similarities between the forms of the ATP7B gene are getting closer as well. As the distance between the species are further apart, differences between the forms of the ATP7B gene and conservation scores are increasing.<br>

Using the aforementioned code, 60 sample single nucleotide mutations with unknown significance were analyzed and a table was created, it can be seen below, which is containing various features of these mutations, including the criticality of the amino acid position in which the mutation occurred, the specific nucleotide replacements, and the resulting change in the polarity/charge of the replaced amino acid. This polarity/charge change was classified into four categories: polar, non-polar, positively charged, and negatively charged. It is important to note that even a mutation in a conserved region, as calculated by the "pos_conserv.py" code, may not be pathogenic if the replaced amino acids are of the same polarity/charge category. Of the 60 mutations resulting in amino acid replacement, 26 were calculated to potentially have critical significance, while 12 involved a change in polarity/charge. As an example, a reported mutation (Leu1371Arg) that had occurred in the 1371st amino acid position in the original sequence, which is a Leucine to Arginine substitution, changes this position’s charge from non-polar to positively charged. “Ser1369Leu” demonstrate a substitution of Serine as Leucine and the charge changes from polar to non-polar. Another substitution, Gly1355Asp, is an example of non-polar to negatively charged amino acid change. All these mutations have a higher probablity of affecting the protein’s 3D structure since there is a change in the amino acids’ charge which is crucial for a protein’s folding mechanism. <br><br>



|  Name         | Amino Acid Change | Position | Polarity/Charge Change | Position With Gaps | Nucleotide Change | Significance |
|   --------    | ----- |  -------  |  ---  | ----  |  ---------  | --- |
|  Tyr1464Ser   |  Y-S  |  1464     |  P_P  | 2555  | 4391A>C     | No| 
|  Gln1463Arg   |  Q-R  |  1463     |  P_+  | 2554  | 4388A>G     | No| 
|  Gln1463Ter   |  Q-*  |  1463     |  P_*  | 2554  | 4387C>T     | No| 
|  Asp1460Gly   |  D-G  |  1460     |  -_N  | 2543  | 4379A>G     | No| 
|  Leu1454Pro   |  L-P  |  1454     |  N_N  | 2534  | 4361T>C     | No| 
|  Asp1450Asn   |  D-N  |  1450     |  -_P  | 2471  | 4383G>A     | No| 
|  Ala1443Ser   |  A-S  |  1443     |  N_P  | 2461  | 4327G>T     | No| 
|  Arg1440Gln   |  R-Q  |  1440     |  +_P  | 2453  | 4319G>A     | No| 
|  Arg1440Trp   |  R-W  |  1440     |  +_N  | 2453  | 4318C>T     | No| 
|  Ser1439Pro   |  S-P  |  1439     |  P_N  | 2452  | 4315T>C     | No| 
|  Ser1432Phe   |  S-F  |  1432     |  P_N  | 2443  | 4295C>T     | No| 
|  Leu1430Met   |  L-M  |  1430     |  N_N  | 2441  | 4288C>A     | No| 
|  Ser1429Leu   |  S-L  |  1429     |  P_N  | 2440  | 4286C>T     | No| 
|  Tyr1424Cys   |  Y-C  |  1424     |  P_N  | 2435  | 4271A>G     | No| 
|  Ser1423Asn   |  S-N  |  1423     |  P_P  | 2433  | 4286G>A     | No| 
|  Arg1411Gln   |  R-Q  |  1411     |  +_P  | 2416  | 4232G>A     | No| 
|  Ser1398Thr   |  S-T  |  1398     |  P_P  | 2403  | 4192T>A     | Yes| 
|  Pro1394Leu   |  P-L  |  1394     |  N_N  | 2399  | 4181C>T     | No| 
|  Met1392Thr   |  M-T  |  1392     |  N_P  | 2397  | 4175T>C     | No| 
|  His1389Arg   |  H_R  |  1389     |  +_+  | 2394  | 4166A>G     | No| 
|  Ala1388Val   |  A-V  |  1388     |  N_N  | 2393  | 4163C>T     | No| 
|  Leu1381Val   |  L-V  |  1381     |  N_N  | 2384  | 4141C>G     | No| 
|  Tyr1376Ser   |  Y-S  |  1376     |  P_P  | 2379  | 4127A>C     | Yes| 
|  Cys1375Ser   |  C-S  |  1375     |  N_P  | 2378  | 4124G>C     | No| 
|  Leu1373Pro   |  L-P  |  1373     |  N_N  | 2376  | 4118T>C     | No| 
|  Gln1372Lys   |  Q-K  |  1372     |  P_+  | 2375  | 4114C>A     | No| 
|  Gln1372Glu   |  Q-E  |  1372     |  P_-  | 2375  | 4114C>G     | No| 
|  Leu1371Arg   |  L-R  |  1371     |  N_+  | 2374  | 4112T>G     | Yes| 
|  Ser1369Leu   |  S-L  |  1369     |  P_N  | 2372  | 4106C>T     | Yes| 
|  Leu1368Val   |  L-V  |  1368     |  N_N  | 2371  | 4102C>G     | No| 
|  Val1364Leu   |  V-L  |  1364     |  N_N  | 2367  | 4090G>C     | Yes| 
|  Met1359Ile   |  M-I  |  1359     |  N_N  | 2362  | 4077G>T     | Yes| 
|  Ala1357Val   |  A-V  |  1357     |  N_N  | 2360  | 4070C>T     | Yes| 
|  Ser1356Pro   |  S-P  |  1356     |  P_N  | 2359  | 4066T>C     | Yes| 
|  Gly1355Asp   |  G-D  |  1355     |  N_-  | 2358  | 4064G>A     | Yes| 
|  Gly1355Val   |  G-V  |  1355     |  N_N  | 2358  | 4064G>T     | Yes| 
|  Gly1355Ser   |  G-S  |  1355     |  N_P  | 2358  | 4064G>A     | Yes| 
|  Trp1353Arg   |  W-R  |  1353     |  N_+  | 2356  | 4057T>C     | Yes| 
|  Val1349Leu   |  V-L  |  1349     |  N_N  | 2352  | 4045G>C     | No| 
|  Met1344Leu   |  M-L  |  1344     |  N_N  | 2347  | 4030A>C     | No| 
<br>

__Fig.3__ :Table of the examined VUS sequences showing their amino acid substitutions, locations, affinity changes, and whether it is significant or not. Significance of an aminoacid means the pathogenicity of the substitution. Rest of the table can be seen on Github. <br><br>

Regions in which nucleotide replacements have a critical significance are thought be more conserved than the regions in which nucleotide replacements don’t have a critical significance.  This emphasizes that the region in which there is a critical significance of the nucleotide replacement is crucial for the survival of the species. Since the region is crucial for the survival, the mutations in that region are not transferred into new generations and they are not shown in the phylogenetic tree. Considering these information, we can conclude that mutations occurring in these critical, conserved regions have more probability of being pathogenetic. <br>

On the other hand, if the nucleotide replacement  in a conserved region causes replacement of amino acids with the same polarity, for example if both of them are non-polar, the change is thought to have less effect on the survival of the species. Considering these information, we can conclude that mutations occurring in these regions have more probability of being benign. <br>

![Conservation Graph](/conservation_graph.png) <font size="5">__Fig.4__ :Graph of conservation scores of the positions in the aligned sequences. x-axis is the position number, y-axis is the conservation score calculated by entrophy. </font>

A graph has been plotted to depict the relationship between the amino acid position and the entropy score. This graph, which can be seen above, illustrates regions of relatively low entropy, including some near-ground-level regions, which may suggest the crucial role of these regions in the healthy functioning of ATP7B. However, it is important to note that this graph provides a general overview of the distribution of conserved regions and may not be sufficient to fully understand the importance of these regions in maintaining the protein's primary function and its role in vital biological processes, such as copper transport.



## Discussion

In this study, the ATP7B gene, a known genetic determinant of Wilson's disease in humans, was analyzed to identify conserved amino acid positions among species carrying the homologous ATP7B gene. The conservation rate of these positions was calculated using an entropy calculation method that allows for the quantification of amino acid changes of unknown significance, which was adopted from a previously published source. A phylogenetic tree was also constructed to visualize the evolutionary relationships between species and the similarity of their ATP7B gene sequences. A code was utilized to analyze 60 sample single nucleotide mutations and generate a table featuring various characteristics of these mutations, including the criticality of the amino acid position in which the mutation occurred, the specific nucleotide replacements, and the resulting change in polarity and charge of the replaced amino acid. These mutations are more likely to affect the protein's three-dimensional structure due to the changes in amino acid charge, which plays a crucial role in the protein's folding mechanism. Regions with critical significance for nucleotide replacement are thought to be more conserved and potentially more important for species survival, and mutations in these regions are more likely to be pathogenic. Conversely, mutations involving amino acid replacements with the same polarity in conserved regions are more likely to be benign. <br>

## Materials and Methods

The initiation of the project is done with finding a rare genetic disease that is inherited in a Mendelian type fashion. Wilson’s disease (WD) is selected for the topic of the project as it is an autosomal recessive disease and meets the criteria for it. As the Wilson’s disease is caused by the functional changes in the copper-transporting ATPase 2, coded by the ATP7B gene, the amino acid sequence of the protein is derived from UniProt database. Then, the homologous sequences of the copper-transporting ATPase 2 are found with BLAST, via using blastp algorithm against 1000 sequences. Followingly, the sequences retrieved from BLAST are aligned using Muscle algorithm in MEGA11 software. The Newick tree of the aligned homologous sequences is drawn with the same software using Neighbor-joining method. The tree is then visualized using FigTree, with midpoint rooting to identify the protein family according to its evolutionary history. <br> <br>

![Colored Tree](/tree_colored.png) <font size="5">__Fig.5__ :Blue titles indicate *Homo sapiens*, yellow clade shows the clade that all the *Homo sapiens* are included, and green clade shows the clade that contains only *Homo sapiens*. </font>




In order to calculate the conservation scores of the amino acid positions in the aligned sequence, a Python code is written. Entropy-based measure algorithm is used to calculate the conservation scores of each position, which uses the formula 
$$\sum_{a = 1}^{20} f_a(i)*\ln(f_a(i))$$
i being the position number (Pei et al., 2001). The code calculates the entropy of the each positions for the aligned sequence and stores it. The Variances of Unknown Significance with single amino acid changes are derived from the ClinVar database with the information on the position. The original position of the change is converted to position on the aligned sequence by adding gaps using MEGA11. The code asks for the position with gaps and returns whether a change in the inputted location is critical (highly possible to be pathogenic) or not, according to the conservation score thresholds. <br> <br>



## References

Forbes, J. R., & Cox, D. W. (1998). Functional characterization of missense mutations in ATP7B: Wilson disease mutation or normal variant?. *The American Journal of Human Genetics, 63*(6), 1663-1674.

Pei, J., & Grishin, N. V. (2001). AL2CO: calculation of positional conservation in a protein sequence alignment. *Bioinformatics, 17*(8), 700-712.

*ClinVar Database.* (2022) https://www.ncbi.nlm.nih.gov/clinvar/?gr=1&term=atp7b%5Bprotein%5D 

BLAST Database. (2022) https://blast.ncbi.nlm.nih.gov/Blast.cgi# 



